tests/conftest.py

```diff
@@ -38,14 +40,16 @@ def start():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def start_server(request):
+def start_server(
+    request,
+):  # Input must exist due to 'fixture'. pylint:disable=unused-argument
```

Comment:
Side note: Consider installing [pylint-pytest](https://pypi.org/project/pylint-pytest/) (not to be confused with pytest-pylint ;-)) to avoid having to suppress this kind of pylint violation.

Reply:
Let's do that in a separate commit as I have a lot to do with ETOS now :)